apps/src/applab/applab.js

```diff
@@ -1305,12 +1306,10 @@ Applab.showConfirmationDialog = function(config) {
       '<p>' + config.text + '</p>';
 
   var buttons = document.createElement('div');
-  buttons.innerHTML = require('../templates/buttons.html.ejs')({
-    data: {
-      confirmText: commonMsg.dialogOK(),
-      cancelText: commonMsg.dialogCancel()
-    }
-  });
+  React.render(React.createElement(Buttons, {
```

Comment:
Make sure things place nice with React v0.14. I think this needs to be ReactDOM.render now